src/frontend/scss/objects/_conversation_topic_detail.scss

```diff
@@ -44,6 +44,27 @@
         padding-top: 5px;
         border-top: 1px dashed $gray-200;
       }
+
+      blockquote {
+        background: #f9f9f9;
+        border-left: 10px solid #ccc;
+        margin: 1.5em 10px;
+        padding: 0.5em 10px;
+        quotes: '\201C''\201D''\2018''\2019';
+      }
+
+      blockquote:before {
+        color: #ccc;
+        content: '';
+        font-size: 4em;
+        line-height: 0.1em;
+        margin-right: 0.25em;
+        vertical-align: -0.4em;
+      }
```

Comment:
Since, you do not set any content to `:before`, this pseudo-element just add a left blank to the first line of the blockquote. So why not just use `text-indent` to do that ?